sql/src/main/java/io/druid/sql/calcite/expression/Expressions.java

```diff
@@ -19,92 +19,130 @@
+      .put(SqlStdOperatorTable.CHARACTER_LENGTH, "strlen")
+      .put(SqlStdOperatorTable.CONCAT, "concat")
+      .put(SqlStdOperatorTable.EXP, "exp")
+      .put(SqlStdOperatorTable.DIVIDE_INTEGER, "div")
+      .put(SqlStdOperatorTable.LIKE, "like")
+      .put(SqlStdOperatorTable.LN, "log")
+      .put(SqlStdOperatorTable.LOWER, "lower")
+      .put(SqlStdOperatorTable.LOG10, "log10")
+      .put(SqlStdOperatorTable.POWER, "pow")
+      .put(SqlStdOperatorTable.REPLACE, "replace")
+      .put(SqlStdOperatorTable.SQRT, "sqrt")
+      .put(SqlStdOperatorTable.TRIM, "trim")
+      .put(SqlStdOperatorTable.UPPER, "upper")
       .build();
 
-  private static final Map<SqlTypeName, ExprType> MATH_TYPES;
+  private static final Map<SqlOperator, String> UNARY_PREFIX_OPERATOR_MAP = ImmutableMap.<SqlOperator, String>builder()
+      .put(SqlStdOperatorTable.NOT, "!")
+      .put(SqlStdOperatorTable.UNARY_MINUS, "-")
+      .build();
+
+  private static final Map<SqlOperator, String> UNARY_SUFFIX_OPERATOR_MAP = ImmutableMap.<SqlOperator, String>builder()
+      .put(SqlStdOperatorTable.IS_NULL, "== ''")
+      .put(SqlStdOperatorTable.IS_NOT_NULL, "!= ''")
+      .put(SqlStdOperatorTable.IS_FALSE, "<= 0") // Matches Evals.asBoolean
+      .put(SqlStdOperatorTable.IS_NOT_TRUE, "<= 0") // Matches Evals.asBoolean
+      .put(SqlStdOperatorTable.IS_TRUE, "> 0") // Matches Evals.asBoolean
+      .put(SqlStdOperatorTable.IS_NOT_FALSE, "> 0") // Matches Evals.asBoolean
+      .build();
+
+  private static final Map<SqlOperator, String> BINARY_OPERATOR_MAP = ImmutableMap.<SqlOperator, String>builder()
+      .put(SqlStdOperatorTable.MULTIPLY, "*")
+      .put(SqlStdOperatorTable.MOD, "%")
+      .put(SqlStdOperatorTable.DIVIDE, "/")
+      .put(SqlStdOperatorTable.PLUS, "+")
+      .put(SqlStdOperatorTable.MINUS, "-")
+      .put(SqlStdOperatorTable.EQUALS, "==")
+      .put(SqlStdOperatorTable.NOT_EQUALS, "!=")
+      .put(SqlStdOperatorTable.GREATER_THAN, ">")
+      .put(SqlStdOperatorTable.GREATER_THAN_OR_EQUAL, ">=")
+      .put(SqlStdOperatorTable.LESS_THAN, "<")
+      .put(SqlStdOperatorTable.LESS_THAN_OR_EQUAL, "<=")
+      .put(SqlStdOperatorTable.AND, "&&")
+      .put(SqlStdOperatorTable.OR, "||")
+      .build();
+
+  private static final Map<SqlTypeName, ExprType> EXPRESSION_TYPES;
 
   static {
     final ImmutableMap.Builder<SqlTypeName, ExprType> builder = ImmutableMap.builder();
 
-    for (SqlTypeName type : SqlTypeName.APPROX_TYPES) {
+    for (SqlTypeName type : SqlTypeName.FRACTIONAL_TYPES) {
       builder.put(type, ExprType.DOUBLE);
     }
 
-    for (SqlTypeName type : SqlTypeName.EXACT_TYPES) {
+    for (SqlTypeName type : SqlTypeName.INT_TYPES) {
       builder.put(type, ExprType.LONG);
     }
 
     for (SqlTypeName type : SqlTypeName.STRING_TYPES) {
       builder.put(type, ExprType.STRING);
     }
 
-    MATH_TYPES = builder.build();
+    builder.put(SqlTypeName.BOOLEAN, ExprType.LONG);
```

Comment:
I guess this is because we currently don't support three-valued logic and it will make the future conversion easier. Am I right? If so, please add some comments for this.